spec/models/wiky_spec.rb

```diff
@@ -16,4 +16,15 @@
       expect((wiky.reload).public).to eq(true) 
     end 
   end
-end
+
+  describe ".search_for" do 
```

Comment:
Think about adding another `it` block for any edge cases you can think of. One edge case is what happens when `Wiki.search_for(...)` finds no matching results.